index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bee Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav class="nav">
+                <img src="./ImageP1/hamburger.png" alt="hamburger">
+            </nav>
+        </header>
+        <section class="background">
+        <header class="myheader">
+            <h1> <span class="blue">Hey there. <br>
+                I’m Alex. <br></span>
+               <span class="green"> I’m a software <br>
```

Comment:
- [ ] [OPTIONAL] Please increase the font weight a little bit more for the headings to make them look like as close to figma template as possible

Your design:

<img width="377" alt="Screenshot 2023-03-23 at 9 12 26 AM" src="https://user-images.githubusercontent.com/44798044/227103176-6b4e99a4-db7f-4daf-a986-779f225883c3.png">

Figma design:

<img width="313" alt="Screenshot 2023-03-23 at 9 12 18 AM" src="https://user-images.githubusercontent.com/44798044/227103226-e742dbd3-2121-4bae-888a-b4fac787f51c.png">